test/workers/test_plan.py

```diff
@@ -103,78 +103,78 @@ def forward(self, x):
     pointer_to_result = net(pointer_to_data)
     pointer_to_result.get()
 
+# def test_multiple_workers(workers):
```

Comment:
Is there a reason these tests were removed? Should we replace them with other tests?

Reply:
Yes, these tests require non-plan-creators to build plans
here is our discussion with Marianne:

Marianne Monteiro 
another quick question (take your time to answer these btw): in the current implementation would it be possible to send a plan to a remote worker? apparently in the current implementation to be able to actually run a plan, you need to have direct/indirect access to the blueprint. Indirect access would be some other virtual worker has access to it and ask you to register the objects you need, but this operation, as far as I know, can not be done remotely right now, right?


Marianne Monteiro 
I was able to make plans work (not being a worker), for the 3 first test cases (https://github.com/OpenMined/PySyft/blob/dev/test/workers/test_plan.py) the 3 last ones are not working, because since plans are not workers anymore they are not being hooked, and so the blueprint attribute does not exist and a plan can not be built remotely.

Théo Ryffel 
I'm not sure Iv'e fully understood the question 
To run a plan, you need to have access to a readable_plan, not necessarily the blueprint. But to build it it's right, you need to hold the blueprint.
So it would seem reasonable that the creator of the plan (which has the blueprint) should be the one to build it


Théo Ryffel 
There I agree that the 3 last tests should not be possible now!

Reply:
@iamtrask  @LaRiffle just fixed these tests on PR #2108